### ISSUE TITLE
position of adverb

### ISSUE THREAD

**Issue by danielnaber** [languagetool-org/languagetool#1975]
Problem not detected by LT: "How to properly wash your car", -> "How to wash your car properly" is correct

@MikeUnwalla can you confirm and if so, write a rule for this?

**Comment by MikeUnwalla** [comment]
Some people say that a split infinitive is incorrect. Oxford (and others) do not agree (https://www.lexico.com/en/grammar/split-infinitives), I will make a style rule.

**Comment by danielnaber** [comment]
Well, if the rule wouldn't be on by default because people don't agree, we don't need a rule at all yet, I think?

**Comment by MikeUnwalla** [comment]
I was going to write a rule that is on. If people don't agree, then they can deactivate the rule.

I won't make a rule unless you confirm that you want a rule.

**Comment by danielnaber** [comment]
> If people don't agree, then they can deactivate the rule.

I don't think we should have a rule. Even having to deactivate the rule will annoy some people, plus many people will have a hard time even making the decision.

**Comment by danielnaber** [comment]
BTW, Grammarly also doesn't complain about "How to properly wash your car: ...".
